packages/core/src/lib/utils.ts

```diff
@@ -125,28 +129,39 @@ const hasJSX = (path: Collection<ASTNode>): Boolean => findJSX(path).size() > 0
 // ToDo: Find if the React Component has findComponentDidCatch Method
 const findComponentDidCatchMethod = (
   path: Collection<ASTNode>
-): Collection<ASTNode> => path
-
+): Collection<ASTNode> => {
+  // Filter our path down to a collection of AST nodes that ONLY contains items in the following form: 
+  // ClassBody -> MethodDefinition -> Value -> Key -> KeyName : [fnName]
+  // If the [fnName] === untransformable, then we add it to our modified path collection. 
+  path = path.find(MethodDefinition).filter(element => element.value.key['name'] === 'componentDidCatch');
+  return path;
+}
 // ---------------------------------------------------------------------------
 // Checks if the file has findComponentDidCatch Method
 const hasComponentDidCatchMethod = (path: Collection<ASTNode>): Boolean => {
-  // return findComponentDidCatchMethod(path).size() === 1
-  return false
+  // If our path has 1 or more componentDidCatchMethods, return true
+  return findComponentDidCatchMethod(path).size() > 0;
 }
 
 // ---------------------------------------------------------------------------
 // ToDo: Find if the React Component has findGetDerivedStateFromError Method
 const findGetDerivedStateFromErrorMethod = (
   path: Collection<ASTNode>
-): Collection<ASTNode> => path
+): Collection<ASTNode> => {
```

Comment:
plz run prettier or use editor plugin to fix formatting

Reply:
Formatting like this ok? 

```
const findComponentDidCatchMethod = (
  path: Collection<ASTNode>
): Collection<ASTNode> => {
  return path.find(MethodDefinition).filter(element => element.value.key['name'] === 'componentDidCatch');
}
```

```
const hasComponentDidCatchMethod = (path: Collection<ASTNode>): Boolean => findComponentDidCatchMethod(path).size() > 0

```